Azure-Samples/chat-with-your-data-solution-accelerator
# feat: Text to Speech feature implementation

## Purpose

* user can perform text to speech functionality now in CWYD

## Does this introduce a breaking change?





## How to Test
*  Get the code
*  Deploy in Azure
* Upload documents in Admin page
* Start chatting with Bot
* You can see Text to speech (Speaker) icon for all the responses.

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...
<img width="821" alt="image" src="https://github.com/user-attachments/assets/7af41e9e-5af4-4272-808b-27a06d379464">


## Other Information
